vtex-sites/gatsby.store
# fix: duplicated query at `useTopSearch` hook

## What's the purpose of this pull request?

When trying to use `yarn generate` three errors happened, as shown in the images below:
![Screen Shot 2022-06-09 at 6 56 16 PM](https://user-images.githubusercontent.com/8497187/172953040-12aa8890-9ddf-432b-b2eb-7dce3eed8765.png)

![Screen Shot 2022-06-09 at 6 56 52 PM](https://user-images.githubusercontent.com/8497187/172953034-84d08fb5-0c6c-4ef3-8491-9d8bd8c338cd.png)

This PR intends to fix these errors

## How does it work?

First, I've changed the query unique identifier to `TopSearchSuggestionsQuery` instead of `SearchSuggestionsQuery`, that's being already used at the `useSuggestions` hook.

Then, I've added all asked fields to the query.

After this, `yarn generate` runs without error.

## How to test it?

Try using the `yarn generate` command in your local environment. It's also important to check the preview for any errors at the Search implementation.

## Checklist

<em>You may erase this after checking them all ;)</em>

**Changelog**

**PR Description**

**Documentation**
